### ISSUE TITLE
Unexpected token import while debugging single test

### ISSUE THREAD

**Issue by Steinblock** [maty21/mocha-sidebar#57]
I have the following in my `settings.json`

```
  // Mocha: Glob to search for test files
  "mocha.files.glob": "src/test/**/*.ts",

  // Array of module required to run Mocha. (for example babel-register, coffee-script/register or ts-node/register)
  "mocha.requires": [ "ts-node/register" ],

```
The mocha sidebar shows my tests and runs the just fine.
If I click on debug I get the following error:

```
SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:607:28)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at C:\Users\user\.vscode\extensions\maty.vscode-mocha-sidebar-0.14.20\node_modules\mocha\lib\mocha.js:220:27
    at Array.forEach (<anonymous>)
    at Mocha.loadFiles (C:\Users\user\.vscode\extensions\maty.vscode-mocha-sidebar-0.14.20\node_modules\mocha\lib\mocha.js:217:14)
    at Mocha.run (C:\Users\user\.vscode\extensions\maty.vscode-mocha-sidebar-0.14.20\node_modules\mocha\lib\mocha.js:469:10)
    at Object.<anonymous> (C:\Users\user\.vscode\extensions\maty.vscode-mocha-sidebar-0.14.20\node_modules\mocha\bin\_mocha:404:18)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

In the debug console I can see the following command is executed:

```
node --inspect-brk=24297 ..\..\..\.vscode\extensions\maty.vscode-mocha-sidebar-0.14.20\node_modules\mocha\bin\_mocha src/test/**/*.ts --grep '^Hello function should return hello world$' 
```
As you can see, no arguments are passed to mocha. If I add `--require ts-node/register` the run works. How can I pass arguements to mocha while debugging a single test?

If I create a `mocha.opts` file a directory `test` with `--require ts-node/register` debugging will work. But that's not a good solution since my tests live in `src/test/` not in `test`

**Comment by maty21** [comment]
can you please send sample tks

**Comment by JulZimmermann** [comment]
Same problem here. Mocha is executed without the right parameters for transpiling typescript

**Comment by yehiyam** [comment]
fixed in v0.15.0
